index.html

```diff
@@ -0,0 +1,464 @@
+        /library which gets routed to the existing app instance, enqueued in the
+        page's {{Window/launchQueue}} which, through the assigned
+        {{LaunchConsumer}}, brings the library section of the music player into
+        focus without affecting the current music playback.
+      </p>
+    </section>
+    <section data-link-for="WebAppManifest">
+      <h2>
+        Extension to the Web App Manifest
+      </h2>
+      <p>
+        The following steps are added to the <a data-cite=
+        "!appmanifest#dfn-extension-point">extension point</a> in the steps for
+        <a data-cite="appmanifest#dfn-processing-a-manifest">processing a
+        manifest</a>:
+      </p>
+      <ol>
+        <li>Run the steps for [=processing the launch_handler member=]
+        given [=ordered map=] |json:ordered map| and [=ordered map=]
+        |manifest:ordered map|.
+        </li>
+      </ol>
+      <section>
+        <h3>
+          [=manifest/launch_handler=] member
+        </h3>
+        <p>
+          The <dfn data-dfn-for="manifest">`launch_handler`</dfn> is a dictionary
+          containing configurations for how web app launches should behave.
+        </p>
+        <p class="note">
+          [=manifest/launch_handler=] is a dictionary despite [=manifest/route_to=] being the only member, this is to give room for more members to be added should other types of behaviors be needed in the future.
+        </p>
+        <p>
+          The steps for <dfn>processing the launch_handler member</dfn>, given
+          [=ordered map=] |json:ordered map|, [=ordered map=]
+          |manifest:ordered map|, are as follows:
+        </p>
+        <ol class="algorithm">
+          <li>If |json|["launch_handler"] does not [=map/exist=], return.
+          </li>
+          <li>If the type of |json|["launch_handler"] is not [=ordered map=],
+          return.
+          </li>
+          <li>Set |manifest|["launch_handler"] to a new [=ordered map=].
+          </li>
+          <li>[=Process the `route_to` member=] passing
+          |json|["launch_handler"], |manifest|["launch_handler"].
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h3>
+          [=manifest/route_to=] member
+        </h3>
+        <p>
+          The <dfn data-dfn-for="manifest">`route_to`</dfn> member of the manifest is a [=string=] or list of [=strings=] that declares
+          whether web app launches reuse existing app [=service worker client/window clients=] if any exist and whether or not to navigate their browsing context to the launch's {{LaunchParams/targetURL}} according to a [=route to target=].
+        </p>
+        <p>
+          User agents MAY support only a subset of the [=route to targets=] depending on the constraints of the platform (e.g. mobile devices may not support multiple app instances simultaneously).
+        </p>
+        <p>
+          <p>
+            The <dfn>route to targets</dfn> are as follows:
```

Comment:
Maybe these could be called "route to modes" instead of "targets", since "target" implies an object that receives something (e.g. the clients themselves), whereas these are really behaviour specifiers.

Reply:
Continuing discussion here: https://github.com/WICG/sw-launch/issues/56#issuecomment-1144451505